build-support/bin/generate_travis_yml.py

```diff
@@ -576,16 +581,17 @@ def rust_tests_osx() -> Dict:
     **_RUST_TESTS_BASE,
     "name": "Rust tests - OSX",
     "os": "osx",
-    # Fuse actually works on this image. It hangs on many others.
     "osx_image": "xcode11",
-    "addons": {
-      "homebrew": {
-        "packages": ["openssl"],
-        "casks": ["osxfuse"]
-      }
-    },
+    # We cannot currently run fuse tests in CI, as the osx_image doesn't support it.
+    # Until that changes, no need to install osxfuse.
+    # "addons": {
+    #   "homebrew": {
+    #     "packages": ["openssl"],
+    #     "casks": ["osxfuse"]
+    #   }
+    # },
     "before_install": [
-      './build-support/bin/install_python_for_ci.sh "${PYENV_PY27_VERSION}" "${PYENV_PY36_VERSION}"',
+      './build-support/bin/install_python_for_ci.sh "${PYENV_PY36_VERSION}"',
```

Comment:
+1 for not installing Py27. We do need it for the main test shards because it's necessary for IT subprocesses, but seems unnecessary here.